pkg/plugins/platform/local_test.go

```diff
@@ -534,6 +519,11 @@ func TestCleanupWithoutCompose(t *testing.T) {
 		assert.False(volume.Labels["com.docker.compose.project"] == "ddev"+strings.ToLower(app.GetName()))
 	}
 
+	// Cleanup the global site database dirs
```

Comment:
Actually.. had to back off and not do that. Turned out that other tests depend on the site dir, which is removed by site.Cleanup()